src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,64 @@
-export const Tabs = () => {};
+import React from 'react';
+import classNames from 'classnames';
+
+export interface TabsType {
+  tabs: Tab[];
+  selectedTabId: string;
+  onTabSelected: (arg: Tab) => void;
+}
+
+export type Tab = {
+  id: string;
+  title: string;
+  content: string;
+};
+
+export const Tabs: React.FC<TabsType> = ({
+  tabs,
+  selectedTabId,
+  onTabSelected,
+}) => {
+  const handleTabSelect = (tab: Tab) => {
+    if (tab.id !== selectedTabId) {
+      onTabSelected(tab);
+    }
+  };
+
+  const isTabActive = (tabId: string, index: number) => (
+    tabId === selectedTabId
+    || (!tabs.some(({ id }) => (id === selectedTabId)) && index === 0)
```

Comment:
Hi! This function is needed to pass all tests. If we have `selectedTabId` which does not exist one test crashes. In this case (when `selectedTabId` does not exist in `tabs)` we need to add `is-active` class to the first tab (`index === 0`). 

So, if we have this case from tests, this function checks if this selected id is there in `tabs`. If it is not, and this is the first iteration of `tabs.map()` when `index === 0` this function returns true, the first tab gets class `is-active`, and the function returns. 

In other cases, this function returns true when `tabId === selectedTabId`, and we add `is-active` class to the existing selected tab. If we remove this check `index === 0` from there, we'll get all tabs `is-active` instead of 'tab-1' in this case with no existing id.

![Знімок екрана 2023-02-09 о 13 13 25](https://user-images.githubusercontent.com/112945779/217929995-9bf63bbb-acf2-4318-a67c-941d1c66952e.png)
![Знімок екрана 2023-02-09 о 13 15 48](https://user-images.githubusercontent.com/112945779/217930001-32a194da-dc4f-4b34-88d3-eb9cd3ab3cb6.png)

Reply:
I think we figure out this point in the chat, so I approve the PR, but try to rewrite this construction. And if you will have any questions, ask in the chat

Reply:
Rewrote it, thank you for the constructive review!